lib/domgen/sync/templates/abstract_master_sync_ejb.java.erb

```diff
@@ -85,7 +85,7 @@ public abstract class <%= data_module.sync.abstract_master_sync_ejb_name %>
 
   protected int count<%= entity.qualified_name.gsub('.','') %>Master( @javax.annotation.Nonnull final java.lang.String mappingSourceCode )
   {
-    return ((java.lang.Number) getEntityManager().createNativeQuery( "<%= j_escape_string("SELECT COUNT(*) FROM #{entity.sync.master_entity.sql.qualified_table_name} WHERE #{entity.sync.master_entity.attribute_by_name(:MappingSource).sql.quoted_column_name} = ?") %>" ).setParameter( 1, mappingSourceCode ).getSingleResult() ).intValue();
+    return ((java.lang.Number) getEntityManager().createNativeQuery( "<%= j_escape_string("SELECT COUNT(*) FROM #{entity.sync.master_entity.sql.qualified_table_name} WHERE #{entity.sync.master_entity.attribute_by_name(:MappingSource).sql.quoted_column_name} = ? AND DeletedAt IS NULL") %>" ).setParameter( 1, mappingSourceCode ).getSingleResult() ).intValue();
```

Comment:
You should use `#{ entity.sync.master_entity.attribute_by_name(:DeletedAt).sql.quoted_column_name}` so it works in all the places where this code will appear